.vscode/extensions.json

```diff
@@ -1,9 +1,9 @@
 {
   "recommendations": [
```

Comment:
So this is sorted based on the name after the . ?

Reply:
Not quite. It's based on the name of the extension as seen in the (recommended) extensions tab. It makes sense to arrange them alphabetically there because that's what most users will interact with, as opposed to alphabetically within this file.

Reply:
I really dont care, just thinking of future maintenance. What order will new addtions be placed in?

Reply:
The order in which they appear in the extensions tab?

Reply:
eehm am i missing something here or is the extensions tab sorted according to the array? 

https://github.com/t3-oss/create-t3-turbo/assets/51714798/a68d63c1-aa8f-4a0f-a87e-fe70268cbece

Reply:
You're correct — the extensions appear in the tab in the order in which they appear in the array. That's why I thought it'd be neat to sort the array so they appear in alphabetical order in the tab. See:

![Screenshot from 2023-12-28 05-33-48](https://github.com/t3-oss/create-t3-turbo/assets/68213317/97910d61-4e6d-417d-9979-55bbc08a8578)

I know it's a minor, inconsequential change, but I thought it'd be a nice touch. Recommended extensions aren't updated regularly anyway, so "maintaining" it won't be a problem.

But if you find it unnecessary, I can revert the changes.

Reply:
So my first comment stands then 😅

> So this is sorted based on the name after the . ?